save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/prograssbar/ReactCircle.kt

```diff
@@ -0,0 +1,68 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+@file:JsModule("react-circle")
+@file:JsNonModule
+
+package com.saveourtool.save.frontend.externals.prograssbar
+
+import react.Component
+import react.PropsWithChildren
+import react.ReactElement
+import react.State
+
+/**
+ * External declaration of [ReactCircleProps] react component
+ */
+@JsName("default")
+external class ReactCircle : Component<ReactCircleProps, State> {
+    override fun render(): ReactElement<ReactCircleProps>?
+}
+
+/**
+ * Props of [ReactCircleProps]
+ */
+external interface ReactCircleProps : PropsWithChildren {
+    /**
+     * Defines the size of the circle.
+     */
+    var size: String
+
+    /**
+     * Defines the thickness of the circle's stroke.
+     */
+    var lineWidth: String
+
+    /**
+     * Update to change the progress and percentage.
+     */
+    var progress: String
+
+    /**
+     * Color of "progress" portion of circle (example: "rgb(76, 154, 255)").
+     */
+    var progressColor: String
+
+    /**
+     * Color of "empty" portion of circle (example: "#ecedf0").
+     */
+    var bgColor: String
+
+    /**
+     * Color of percentage text color (example: "#ecedf0").
+     */
+    var textColor: String
```

Comment:
It seems weird that three variables with three similar names can be divided into two groups: one should be set as hex number, another should be set as `rgb` constructor. It seems strange. Please, double check that there two variants are not replaceable.

Reply:
Yes. Works so and so